src/averageAges.js

```diff
@@ -56,6 +89,46 @@ function calculateWomenAverageAge(people, withChildren) {
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
   // write code here
+  const mothers = [];
+  let kids = [];
+
+  const func = function(person) {
+    if (people.find((el) => person.mother === el.name)) {
+      mothers.push(people.find((el) => person.mother === el.name));
+
+      return true;
+    }
+
+    return false;
+  };
+
+  const alternateFunc = function(person) {
```

Comment:
again, you don't have to create functions for that, try to do it without them

Reply:
I made the functions mostly for readability's sake, but I will try to figure out a different approach